dlp/package.json

```diff
@@ -3,38 +3,33 @@
   "description": "Command-line interface for Google Cloud Platform's Data Loss Prevention API",
   "version": "0.0.1",
   "private": true,
-  "license": "Apache Version 2.0",
+  "license": "Apache-2.0",
   "author": "Google Inc.",
-  "contributors": [
-    {
-      "name": "Ace Nassri",
-      "email": "anassri@google.com"
-    },
-    {
-      "name": "Jason Dobry",
-      "email": "jason.dobry@gmail.com"
-    },
-    {
-      "name": "Jon Wayne Parrott",
-      "email": "jonwayne@google.com"
-    }
-  ],
   "repository": {
     "type": "git",
     "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
   },
-  "scripts": {
-    "test": "ava system-test/*.test.js -c 20 -T 240s"
+  "cloud": {
+    "requiresKeyFile": true,
+    "requiresProjectId": true
   },
   "engines": {
     "node": ">=4.3.2"
   },
+  "scripts": {
+    "system-test": "ava -T 1m --verbose system-test/*.test.js",
+    "test": "npm run system-test"
+  },
   "dependencies": {
-    "google-auth-library": "^0.10.0",
-    "google-auto-auth": "^0.5.2",
+    "google-auth-library": "0.10.0",
```

Comment:
I'm removing it everywhere, as we're moving towards automating dependency updates. We can pin to versions, and let the automation update them. When you pin versions it's easier to get reproducible builds and know what version you have installed.